### PR TITLE
Display graphs in Web Apps

### PR DESCRIPTION
**Pull request by wpride**
Display graphs in the case list and details WIP

screen shot 2017-02-10 at 5 50 15 pm
screen shot 2017-02-10 at 5 50 09 pm
screen shot 2017-02-10 at 5 50 04 pm

cross-request: https://github.com/dimagi/formplayer/pull/283

### CODE REVIEW

**Comment by orangejenny** [comment]
this is beautiful

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js [hunk 1]
```diff
@@ -117,12 +117,14 @@ FormplayerFrontend.module("Menus", function (Menus, FormplayerFrontend, Backbone
         getDetailList: function (detailObject) {
             var headers = detailObject.get('headers');
             var details = detailObject.get('details');
+            var styles = detailObject.get('styles');
             var detailModel = [];
             // we need to map the details and headers JSON to a list for a Backbone Collection
             for (var i = 0; i < headers.length; i++) {
                 var obj = {};
-                obj.data = details[i];
+                obj.data = details[i].replace(/\"/g, "'");
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js]: why is this needed?

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js [hunk 2]
```diff
@@ -208,8 +208,12 @@ FormplayerFrontend.module("Menus.Views", function (Views, FormplayerFrontend, Ba
 
         templateHelpers: function () {
             var appId = Util.currentUrlToObject().appId;
+            var data = this.options.model.get('data');
+            for (var i =0; i < data.length; i++) {
+                data[i] = data[i].replace(/\"/g, "'");
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js]: seems like we do this twice.

also nit this is a good use case for map

**Review by benrudolph — COMMENTED** [review]
sweet looking good!

**Comment by benrudolph** [comment]
One thing I would suggest is to style the iframes so that you can't see their borders: http://stackoverflow.com/a/65052/835696

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js (hunk 1 above)
Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js]: Added this because having the double quotes in the payload was messing up the javascript parsing

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js (hunk 2 above)
Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js]: Ben you know I hate maps

**Comment by orangejenny** [comment]
Is the feedback on this blocking? It looks fairly minor, same for https://github.com/dimagi/formplayer/pull/283 . Ali & I would be *really* excited to see this live, even if it's a little bit imperfect.

**Comment by benrudolph** [comment]
ah ok didn't realize there was urgency to this, can merge now

**Comment by orangejenny** [comment]
Awesome, thank you!
